dev-qt/qt5/qt5-5.6.1.recipe

```diff
@@ -322,10 +322,10 @@ INSTALL()
 	mv -f $libDir/libQt5QmlDevTools* $developLibDir/
 	mv -f $libDir/libQt5Bootstrap* $developLibDir/
 	mv -f $libDir/libQt5PlatformSupport* $developLibDir/
-	mv -f $libDir/libQt5OpenGLExtensions* $developLibDir/	
+	mv -f $libDir/libQt5OpenGLExtensions* $developLibDir/
 
-	devCommands="assistant designer linguist lconvert lrelease lupdate 
-		pixeltool qcollectiongenerator qdbusviewer qdbus qhelpconverter
+	devCommands="assistant designer linguist lconvert lrelease lupdate
+	pixeltool qcollectiongenerator qdbusviewer qdbus qhelpconverter
```

Comment:
Oops, I've just noticed you ate one tab in line 328, which starts with the word pixeltool.
Could you please insert it back, so that this line starts with 2 tabs? Thanks again!